libmscore/sym.cpp

```diff
@@ -6274,11 +6275,13 @@ void initScoreFonts()
             }
       for (oldName i : oldNames)
             Sym::lonhash.insert(i.name, SymId(i.symId));
-      QFont::insertSubstitution("MScore Text",    "Bravura Text");
-      QFont::insertSubstitution("Gootville Text", "Bravura Text");
-      QFont::insertSubstitution("ScoreFont",      "Bravura Text");
-      QFont::insertSubstitution("MuseJazz Text",  "Bravura Text");
-      QFont::insertSubstitution("Petaluma Text",  "Bravura Text");
+      QFont::insertSubstitution("Leland Text",    "Bravura Text");
+      QFont::insertSubstitution("Bravura Text",   "Leland Text");
+      QFont::insertSubstitution("MScore Text",    "Leland Text");
+      QFont::insertSubstitution("Gootville Text", "Leland Text");
+      QFont::insertSubstitution("ScoreFont",      "Leland Text");
+      QFont::insertSubstitution("MuseJazz Text",  "Leland Text");
+      QFont::insertSubstitution("Petaluma Text",  "Leland Text");
```

Comment:
Pataluma should better fallback to MuseJazz (first, that then to Lelland and that to Bravura)

Reply:
@Tantacrul and @oktophonie approved that proposal, thx